content/en/boilerplates/verify-crds.md

```diff
@@ -1,6 +1,6 @@
 Verify that all `23` Istio CRDs were committed to the Kubernetes api-server using the following command:
 
 {{< text bash >}}
-$ kubectl get crds | grep 'istio.io' | wc -l
+$ kubectl get crds -n istio-system | grep 'istio.io' | wc -l
```

Comment:
The command works with or without the `-n` option, because CRDs are "virtually" in every namespace, regardless of which namespace they are created in.

Reply:
@frankbu So can we close this PR?